data/config

```diff
@@ -22,6 +22,9 @@ ExitCommand: killall Hyprland
 # The folder, where the battery sensors reside
 BatteryFolder: /sys/class/power_supply/BAT1
 
+# The partion to monitor in disk sensor (default to "/")
+# DiskPartition: /home
```

Comment:
Can you uncomment and change it to /
I have the convention to keep defaults in ```data/config```. 
Commented out should only be used for configs, that don't have a default/default is "".